doc/example/petab_import.ipynb

```diff
@@ -128,6 +128,11 @@
     "print(\"Model states:    \", list(model.getStateIds()), \"\\n\")"
    ]
   },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "The AMICI model can be used to directly simulate the model and customize outputs such as states, sensitivities, and observables. For more details, see the `amici notebook`. We want to integrate this simulation model into an objective function we can repeatedly call during optimization."
+  },
```

Comment:
I don't find this reference particularly helpful here. That other notebook is mostly irrelevant to PEtab users. I'd rather remove the whole `Import model to AMICI` section here, unless it's extended.